File: Data Structures & Algorithms/duplicate-integer/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {boolean}
     */
    hasDuplicate(nums) {
        let objMap = {};
        for(let num of nums ) {
            if(objMap[num]) {
                return true;
            } else {
                objMap[num] = 1;
            }
        }
        return false;
    }

}

File: Data Structures & Algorithms/maximum-subarray/submission-0.py
class Solution:
    def maxSubArray(self, nums: List[int]) -> int:
        meh = 0
        msf = -math.inf
        for num in nums:
            meh = meh + num
            if meh < num:
                meh = num
            if msf < meh:
                msf = meh
        return msf
